pkg/alertmanager/statefulset.go

```diff
@@ -715,6 +717,27 @@ func makeStatefulSetSpec(a *monitoringv1.Alertmanager, config Config, tlsAssetSe
 		minReadySeconds = int32(*a.Spec.MinReadySeconds)
 	}
 
+	operatorInitContainers = append(operatorInitContainers,
+		operator.CreateConfigReloader(
+			"init-config-reloader",
+			operator.ReloaderConfig(config.ReloaderConfig),
+			operator.ReloaderRunOnce(),
+			operator.LogFormat(a.Spec.LogFormat),
+			operator.LogLevel(a.Spec.LogLevel),
+			operator.WatchedDirectories(watchedDirectories),
+			operator.VolumeMounts(configReloaderVolumeMounts),
+			operator.Shard(-1),
```

Comment:
This isn't needed for Alertmanager but I see that you've copied it from the existing reloader config container so it can be cleaned up in a following PR as well.
```suggestion
```

Reply:
Yeah, thought about that. Just wanted to keep as close to the sidecar reloader config as possible